startup/00-base.py

```diff
@@ -63,6 +63,15 @@ def _load(file):
 # PersistentDict will create the directory if it does not exist
 RE.md = PersistentDict(runengine_metadata_dir)
 
+data_session = RE.md.get("data_session")
+if data_session is None:
+    raise ValueError("Data session is None!")
+
+cycle = '2023-3' # TODO: don't hardcode this
+
+proposal_dir = f"/nsls2/data/chx/proposals/{cycle}/{data_session}"
+Path(proposal_dir).mkdir(parents=True, exist_ok=True)
```

Comment:
This Python process will never have permission to make directories here. You can assume the directory has been created.